apps/cli/tsconfig.json

```diff
@@ -9,13 +9,18 @@
     "target": "ES2021",
     "sourceMap": true,
     "outDir": "./dist",
-    "baseUrl": "./",
+    "baseUrl": ".",
+    "paths": {
+      "@/*": ["./src/*"]
```

Comment:
cant we replace `@/*` with `@app`?

Reply:
This is a very standard format for absolute path